src/pages/AddServers.js

```diff
@@ -163,11 +257,13 @@ class AddServers extends BaseUpdateWizardPage {
         <div className='wizard-content unlimited-height'>
           {this.isValidToRenderServerContent() && this.renderAddPage()}
         </div>
-        {this.state.errors && this.renderErrorBanner()}
+        {this.renderDeployConfirmModal()}
+        {this.renderValidationErrorModal()}
+        {this.state.errorBanner && this.renderGetDeployedServerErrorBanner()}
```

Comment:
"renderGetDeployedServerErrorBanner" is descriptive, if perhaps a bit long ;-)

Reply:
k, will fix